packages/components/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Unreleased
 -   `BorderControl`: Apply proper metrics and simpler text ([#53998](https://github.com/WordPress/gutenberg/pull/53998)).
+-   `FormTokenField`: Update styling for consistency and increased visibility ([#54402](https://github.com/WordPress/gutenberg/pull/54402)).
```

Comment:
Let's also move this entry in a sub-section (I'd say "Enhancements" ?)

Reply:
Done in https://github.com/WordPress/gutenberg/pull/54402/commits/5695dabc9d95c1700a771cbdd99daad9f6bb6b27.